homeassistant/components/sensor/strings.ebusd.json

```diff
@@ -0,0 +1,6 @@
+{
+  "state": {
+    "day": "Day",
```

Comment:
This is not a sensor but a binary sensor (two states). Also, I don't see a valid use case that a user would use ebusd to get if it's currently day or night. So let's remove it.

Reply:
Operative mode have four values: on, off, day, night. It also update automatically like most sensors so you will see the change in history and create automations. So I will not remove it